term/src/input.rs

```diff
@@ -50,7 +52,7 @@ pub struct MouseEvent {
 #[derive(Debug, Clone)]
 pub struct LastMouseClick {
     pub button: MouseButton,
-    position: (usize, i64),
+    position: (usize, i64, usize, usize),
```

Comment:
Stylistically speaking, big tuples are hard to grok.

Would you mind turning this tuple into a struct, and having `LastMouseClick::new` and `LastMouseClick::add` use that type for the position info?

Something like this should make it difficult to misinterpret the meaning of the fields in a future pass over the code:

```rust
struct ClickPosition {
   column: usize,
   row: i64,
   x_pixel_offset: usize,
   y_pixel_offset: usize,
}
```